src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -138,12 +138,18 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
 
   const schedule: ScheduleDay = useMemo(() => {
     const startOfSelectedDay = addDays(firstDayOfSchedule, selectedDayIndex);
+
     const daysEvents = relatedVenueEvents
       .filter(isEventWithinDate(startOfSelectedDay))
       .map(prepareForSchedule(startOfSelectedDay, userEventIds));
 
     const locatedEvents: LocatedEvents[] = Object.entries(
-      groupBy(daysEvents, buildLocationString)
+      groupBy(
+        daysEvents.filter(
+          (event) => !event.venueId.match(REMOVE_EVENTS_FROM_VENUE)
+        ),
+        buildLocationString
+      )
     ).map(([group, events]) => ({
```

Comment:
What's the intent/need for this change here?

Reply:
The aim here is to remove ~3900 posterpage-linked events from appearing in the venue rows of the schedule (unless they are 'saved/bookmarked' in the posterhall, in which case they will appear in the personalized schedule)

Reply:
@mike-lvov This doesn't seem like a generalisable solution, so we might end up needing to merge this directly to `env/ohbm` instead of our main code branch. Thoughts?